crates/wash-cli/src/dev.rs

```diff
@@ -434,7 +437,7 @@ impl DependencySpec {
 
     /// Convert to a component that can be used in a [`Manifest`] with a given suffix for uniqueness
     fn generate_component(&self, suffix: &str) -> Result<Component> {
-        let name = format!("{}-{}", suffix, self.name());
+        let name = format!("{}-dep-{}", suffix, self.name());
```

Comment:
dep or dev?

Reply:
`dep` as in dependency!

This comes up when your component is actually named `messaging-nats` :)

Reply:
we've essentially moved the problem (now if your component was named `dep-messaging-nats` it might fail), but... I think that case is practically pretty rare